OSpackages.sh

```diff
@@ -0,0 +1,399 @@
+#!/bin/bash
+set -eu -o pipefail
+#
+# Install primary OS packages into chroot-env
+#
+#######################################################################
+PROGNAME=$(basename "$0")
+CHROOTMNT="${CHROOT:-/mnt/ec2-root}"
+DEBUG="${DEBUG:-UNDEF}"
+FIPSDISABLE="${FIPSDISABLE:-UNDEF}"
+MINXTRAPKGS=(
+      chrony
+      cloud-init
+      cloud-utils-growpart
+      dhcp-client
+      dracut-config-generic
+      firewalld
+      gdisk
+      grub2-pc-modules
+      grub2-tools
+      grub2-tools-minimal
+      grubby
+      kernel
+      kexec-tools
+      lvm2
+      rng-tools
+      unzip
+   )
+EXCLUDEPKGS=(
+      aic94xx-firmware
+      alsa-firmware
+      alsa-tools-firmware
+      biosdevname
+      iprutils
+      ivtv-firmware
+      iwl100-firmware
+      iwl1000-firmware
+      iwl105-firmware
+      iwl135-firmware
+      iwl2000-firmware
+      iwl2030-firmware
+      iwl3160-firmware
+      iwl3945-firmware
+      iwl4965-firmware
+      iwl5000-firmware
+      iwl5150-firmware
+      iwl6000-firmware
+      iwl6000g2a-firmware
+      iwl6000g2b-firmware
+      iwl6050-firmware
+      iwl7260-firmware
```

Comment:
recently had to add `iwl7265-firmware` here, for RHEL builds... 

* https://github.com/plus3it/AMIgen7/pull/94/files

Reply:
/shrug 

Some drivers/firmware are already deprecated in EL8. My include/exclude lists had to be specifically set for EL8's needs.